source/_components/insteon_local.markdown

```diff
@@ -35,7 +35,7 @@ Configuration variables:
 
 ### {% linkable_title Full configuration %} 
 
-The `insteon_local` component currently supports both lights (dimmers) and switches. A full configuration may look like so:
+The `insteon_local` component currently supports both lights (dimmers) and switches, fans are now supported as well. With the recent update there is no longer any need to add the light, switch, or fans under the configuration. 
```

Comment:
Rephrase this sentence to just describe the code as it will exist, not describing the code changes that are currently being introduced. Probably only needs to say "The `insteon_local` component currently supports lights (dimmers), switches and fans".